test/support/cluster/orchestrator.rb

```diff
@@ -17,6 +17,10 @@ def initialize(node_addrs, timeout: 30.0)
     @timeout = timeout
   end
 
+  def restart_cluster_nodes
+    system("make --no-print-directory start_cluster > /dev/null 2>&1")
```

Comment:
It's better to pass the command arguments separately to avoid creating a subshell:

```ruby
system('make', '--no-print-directory', 'start_cluster', out: File::NULL, err: File::NULL)
```